catalog-info.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: kibana
+  description: Kibana is a user interface that lets you visualize your Elasticsearch data and navigate the Elastic Stack.
+
+  annotations:
+    backstage.io/source-location: url:https://github.com/elastic/kibana/tree/main
+    github.com/project-slug: elastic/kibana
+    github.com/team-slug: elastic/kibana-tech-leads
+    buildkite.com/project-slug: elastic/kibana
+
+  tags:
+    - typescript
+    - javascript
+    - dashboards
+    - metrics
+    - visualizations
+    - observability
+
+  links:
+    - title: Documentation
+      url: https://www.elastic.co/guide/en/kibana/current/index.html
+
+spec:
+  type: service
+  owner: group:kibana-tech-leads
+  lifecycle: production
+
+---
+
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: kibana-tests-pipeline
+  description: Definition of the kibana pipeline
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/kibana-tests
+spec:
+  type: buildkite-pipeline
+  owner: group:kibana-tech-leads
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana-tests
+      description: Pipeline that tests the service integration in various environments
+    spec:
+      repository: elastic/kibana
+      pipeline_file: ./.buildkite/pipelines/pipeline.kibana-tests.yaml
+      provider_settings:
+        trigger_mode: none
+      teams:
+        security-engineering-productivity:
+          access_level: MANAGE_BUILD_AND_READ
+        fleet:
+          access_level: MANAGE_BUILD_AND_READ
+        kibana-tech-leads:
+          access_level: MANAGE_BUILD_AND_READ
+        kibana-core:
+          access_level: MANAGE_BUILD_AND_READ
+        cloud-tooling:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
```

Comment:
What qualifies a team to be listed here and what does the team then get? I'm guessing @elastic/kibana-operations should probably be added to this list.

Reply:
With a project of this scale however, I believe that this permission should actually be narrowed down in the future.
We need a finer control over what certain people are able to access. 
This is in particular important for manual verification steps. 
In case a single teams wants to have a manual testing phase in an environment, none else outside of that team should be able to unlock the blocking. 

I think in the future, the `pipeline.kibana-tests.qa` would become the proxy pipeline that just calls each team pipeline which has again different settings . But lets see, a bit of refactoring shouldn't take to long in the future.

But generally, right now:
A benefit of being listed here with more then read is that you can trigger on demand pipelines agains whatever environment to get a fast understanding of the deployed version health. 
And you can fail an automated release process by canceling a build

Reply:
Theses are the folks who can kick off and manipulate the pipeline.  I will add kibana-operations.

I think I should remove `MANAGE` and replace with `BUILD_AND_READ` for everyone else and let `kibana-operations` have MANAGE in case they need to edit the pipeline in bk for whatever reason.  Manage just gives edit pipeline perms in bk ui.

https://buildkite.com/docs/pipelines/permissions#permissions-with-teams-pipeline-level-permissions